### DISCUSSION TITLE
1975-12-29-opportunity-knocks/

### DISCUSSION THREAD

**Discussion by giscus** [FanzOfLenaZavaroni/fanzoflenazavaroni.github.io#7] [General]
# 1975-12-29-opportunity-knocks/

Lena appears as last year’s winner, to present the award to the 1975 recipient, The Frank Jennings Syndicate.

https://fanzoflenazavaroni.github.io/1975-12-29-opportunity-knocks/

**Comment by StephenMolloy1** [comment]
https://www.google.co.uk/books/edition/The_Kaleidoscope_British_Christmas_Telev/IJQACAAAQBAJ?hl=en&gbpv=1&dq=Lena+Zavaroni&pg=PA367&printsec=frontcover

In this book, she also appeared a week earlier in a special songwriters show, there was an edition of Opportunity Knocks on the 22nd according to The Television & Radio Database, but I haven't been able to find out anything else as yet.

https://tvrdb.com/listings/1975-12-22

**Reply by FanzOfLenaZavaroni** [reply]
Hi Stephen, pleased to see the committing is working as you may be aware I have had issues with it. Anyway, I'll get on to the subject.

I am aware of this episode of Opportunity Knocks it was a special in which musical compositions competed to win the Opportunity Knocks' Golden Note Award and £500 towards furthering their musical career.

It had a set of judges and was more like a precursor to New Faces than the usual Opportunity Knocks we know.

Info for the episode is on the [British Film Institute (BFI)](http://collections-search.bfi.org.uk/web/Details/ChoiceFilmWorks/150512572) page for the episode along with a link to the cast of the episode.

<hr>

I think you will find that the Kaleidoscope book has given the date (22 December 1975) and title for "Opportunity Knocks Songwriters Show 1975" but somehow has given the cast for the following show of the 29th in which both Lena Zavaroni and Tammy Jones were on.

Makes me wonder if in rereleased versions of the book, this error would have been acknowledged and fixed as well as other errors the book may have had as many books of this type have errors that get fixed on later releases.

**Reply by StephenMolloy1** [reply]
Thanks for clearing this up, I forgot to consider that they might have printed inaccurate information.
